### PR TITLE
Fixes #786: Add a button to toggle displaying all feedback

### LINKED ISSUE #786: Display only 5 feedbacks on a skill and add a button to load more skills
**Actual Behaviour**
A long list of feedbacks is displayed on visiting a skill

**Expected Behaviour**
We should just see some five feedbacks and a button to load more feedbacks.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #786 

Changes: Add a button to toggle displaying all or less feedbacks

Surge Deployment Link: https://pr-795-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
peek 2018-06-20 12-00

### CODE REVIEW

File: src/components/SkillPage/SkillListing.js [hunk 1]
```diff
@@ -265,9 +266,12 @@ class SkillListing extends Component {
   };
 
   saveSkillUsage = (skill_usage = []) => {
+    // eslint-disable-next-line
```
Comment (anshumanv) [inline src/components/SkillPage/SkillListing.js]: Needed else it throws a warning of no return statement in arrow function

File: src/components/SkillUsageCard/SkillUsageCard.js [hunk 2]
```diff
@@ -51,8 +51,11 @@ class SkillUsageCard extends Component {
   render() {
     let totalSkillUsage = 0;
     if (this.props.skill_usage) {
+      // eslint-disable-next-line
```
Comment (anshumanv) [inline src/components/SkillUsageCard/SkillUsageCard.js]: Needed for the same reason stated above

File: src/components/SkillPage/SkillListing.js (hunk 1 above)
Comment (hkedia321) [reply src/components/SkillPage/SkillListing.js]: @anshumanv I think you can pass the eslint by putting a `return null` statement at the end.

Comment (anshumanv) [reply src/components/SkillPage/SkillListing.js]: Yes, it passes that way but null data is not to be returned into the data array.

**Comment by anshumanv** [comment]
@DNS-404 updated.

**Comment by anshumanv** [comment]
Yes, I thought of shifting it below all reviews but then we won't have the functionality to `Show Less` reviews. Still, do you think we should do it this way? @akshatnitd @Akshat-Jain

**Comment by akshatnitd** [comment]
@anshumanv I think you should add it below the feedbacks only. What we can do further is, on click 'Show More', we can open a new page or a Dialog Box and display all the feedbacks there. This is already being implemented by the iOS and Android teams.

**Comment by anshumanv** [comment]
@akshatnitd updated as per the requirements, please review.

**Comment by akshatnitd** [comment]
Closing and opening, since surge deployment failed.

**Pull request merged**